api/src/main/java/com/messagebird/objects/voicecalls/RecordingResponse.java

```diff
@@ -0,0 +1,47 @@
+package com.messagebird.objects.voicecalls;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.Map;
+
+public class RecordingResponse implements Serializable {
+
+    private static final long serialVersionUID = -3645909436942694671L;
+
+    private List<Recording> data;
+    private Map<String, String> _links;
+    private Pagination pagination;
+
+    public List<Recording> getData() {
+        return data;
+    }
+
+    public void setData(List<Recording> data) {
+        this.data = data;
+    }
+
+    public Map<String, String> get_links() {
+        return _links;
+    }
+
+    public void set_links(Map<String, String> _links) {
```

Comment:
This is only ever used internally. I would remove this accessor and instead set this inside the constructor